does not build on s390x and likely other big endian platform

Hi,

We noticed while building the package for Debian that DJVULibre likely has endianness-related issues.

You might find this _interresting_, or state that you don't want to support S390x because there are likely no real-world user.

Greetings :-)

https://buildd.debian.org/status/package.php?p=python-djvulibre&suite=sid

```
Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/.pybuild/cpython3_3.11_djvu/build/tests/test_decode.py", line 531, in test_decode
    self.assertEqual(s, b'\xFF\xFF\xFF\x00' * 4)
AssertionError: b'\x00\xff\xff\xff\x00\xff\xff\xff\x00\xff\xff\xff\x00\xff\xff\xff' != b'\xff\xff\xff\x00\xff\xff\xff\x00\xff\xff\xff\x00\xff\xff\xff\x00'

----------------------------------------------------------------------
Ran 109 tests in 0.091s

FAILED (SKIP=3, errors=2, failures=1)
E: pybuild pybuild:391: test: plugin distutils failed with: exit code=1: cd /<<PKGBUILDDIR>>/.pybuild/cpython3_3.11_djvu/build; python3.11 -m nose -v tests
	rm -fr -- /tmp/dh-xdg-rundir-saNTo7j9
dh_auto_test: error: pybuild --test --test-nose -i python{version} -p "3.12 3.11" returned exit code 13
make[1]: *** [debian/rules:17: override_dh_auto_test] Error 25
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
make: *** [debian/rules:11: build-arch] Error 2
```

--- a/tests/test_decode.py
+++ b/tests/test_decode.py
@@ -525,7 +525,7 @@ class PageJobsTestCase(TestCase):
         pixel_format = PixelFormatRgbMask(0xFF0000, 0xFF00, 0xFF, bpp=32)
         self.assertIs(page_job.render(RENDER_COLOR, (0, 0, 10, 10), (0, 0, 2, 2), pixel_format, 1, buffer), buffer)
         s = buffer.tobytes()
-        self.assertEqual(s, b'\xFF\xFF\xFF\x00' * 4)
+        self.assertEqual(s, (b'\x00\xFF\xFF\xFF' if sys.byteorder == 'big' else b'\xFF\xFF\xFF\x00') * 4)
 
         if sys.version_info >= (3, 3):
             buffer = bytearray(16)
@@ -535,7 +535,7 @@ class PageJobsTestCase(TestCase):
                 memview
             )
             s = bytes(buffer)
-            self.assertEqual(s, b'\xFF\xFF\xFF\x00' * 4)
+            self.assertEqual(s, (b'\x00\xFF\xFF\xFF' if sys.byteorder == 'big' else b'\xFF\xFF\xFF\x00') * 4)
 
 
 class ThumbnailsTestCase(TestCase):

